### ISSUE TITLE
Unresolved reference: decodeBitmap and Cannot infer a type for this parameter. Please specify it explicitly. error while run application 

### ISSUE THREAD

**Issue by DipawaliParmar8dots** [singerdmx/flutter-quill#2281]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/singerdmx/flutter-quill/issues)

### Flutter Quill version

10.7.5

### Steps to reproduce

I have issue after using quill first 2 3 times  when i run the application working fine and then i get this error ...application can not build

```console
 C:\Users\HP\AppData\Local\Pub\Cache\hosted\pub.dev\quill_native_bridge-10.7.6\android\src\main\kotlin\dev\flutterquill\quill_native_bridge\QuillNativeBridgePlugin.kt: (12, 31): Unresolved reference: decodeBitmap
e: C:\Users\HP\AppData\Local\Pub\Cache\hosted\pub.dev\quill_native_bridge-10.7.6\android\src\main\kotlin\dev\flutterquill\quill_native_bridge\QuillNativeBridgePlugin.kt: (111, 63): Unresolved reference: decodeBitmap
e: C:\Users\HP\AppData\Local\Pub\Cache\hosted\pub.dev\quill_native_bridge-10.7.6\android\src\main\kotlin\dev\flutterquill\quill_native_bridge\QuillNativeBridgePlugin.kt: (111, 78): Cannot infer a type for this parameter. Please specify it explicitly.
e: C:\Users\HP\AppData\Local\Pub\Cache\hosted\pub.dev\quill_native_bridge-10.7.6\android\src\main\kotlin\dev\flutterquill\quill_native_bridge\QuillNativeBridgePlugin.kt: (111, 81): Cannot infer a type for this parameter. Please specify it explicitly.
e: C:\Users\HP\AppData\Local\Pub\Cache\hosted\pub.dev\quill_native_bridge-10.7.6\android\src\main\kotlin\dev\flutterquill\quill_native_bridge\clipboard\ClipboardImageHandler.kt: (11, 31): Unresolved reference: decodeBitmap
e: C:\Users\HP\AppData\Local\Pub\Cache\hosted\pub.dev\quill_native_bridge-10.7.6\android\src\main\kotlin\dev\flutterquill\quill_native_bridge\clipboard\ClipboardImageHandler.kt: (187, 24): Unresolved reference: decodeBitmap
e: C:\Users\HP\AppData\Local\Pub\Cache\hosted\pub.dev\quill_native_bridge-10.7.6\android\src\main\kotlin\dev\flutterquill\quill_native_bridge\clipboard\ClipboardImageHandler.kt: (187, 39): Cannot infer a type for this parameter. Please specify it explicitly.
e: C:\Users\HP\AppData\Local\Pub\Cache\hosted\pub.dev\quill_native_bridge-10.7.6\android\src\main\kotlin\dev\flutterquill\quill_native_bridge\clipboard\ClipboardImageHandler.kt: (187, 42): Cannot infer a type for this parameter. Please specify it explicitly.
```

### Expected results

application build without any issue

### Actual results

get Unresolved reference: decodeBitmap error 

### Additional Context

<details>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

<details><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by EchoEllet** [comment]
Did you included `quill_native_bridge` in your `pubspec.yaml`? This log has changed are included in #2230 though not merged yet, we haven't updated the minimum version of `quill_native_bridge` in `flutter_quill` so those shouldn't be an issue.

Does using older versions of `flutter_quill`

@singerdmx @CatHood0 @AtlasAutocode This happened in #2280, #2264, and other issues, and confirm we have an issue, apps using `flutter_quill` are automatically using the latest versions of dependencies, I still don't get why latest version of [flutter_inappwebview](https://pub.dev/packages/flutter_inappwebview) is being used in my outdated branch when we updated the `pubspec.yaml` of `flutter_quill_extensions` 3 months ago.

Just to confirm, I never updated version of `quill_native_bridge` in `flutter_quill`, and this whole log is not an issue even in my local branch, but it's outside of the branch since the project is using latest version of `quill_native_bridge` when related changes of the branch is not published and is not merged either, it also happened with other dependencies.

@DipawaliParmar8dots Does using version `10.6.6` fix the issue for you? If yes, can you slowly increase the version one after the another until reaching `10.7.5` and see which version have the issue?

**Comment by EchoEllet** [comment]
@singerdmx We need to avoid publishing new releases until this issue is fixed, it causing more issues.

**Comment by EchoEllet** [comment]
It seems that I have missed that we're using the caret constraint (^). Still doesn't answer my question.

Should be fixed in [10.7.7](https://github.com/singerdmx/flutter-quill/releases/tag/v10.7.7).

@DipawaliParmar8dots Can you try the new version and see if the issue remain

```yaml
flutter_quill: ^10.7.7
```
